ClientApp/src/app/Validators/MatchRevealedValidator.ts

```diff
@@ -0,0 +1,15 @@
+import { FormGroup, ValidationErrors, ValidatorFn } from '@angular/forms';
+
+export function MatchRevealedValidator(formControlName: string, formControlNameToCompare: string): ValidatorFn{
```

Comment:
Not sure what MatchRevealedValidator actually means. Can you think of a better name?